### DISCUSSION TITLE
Is there any way to nest a few popovers in dropdown?

### DISCUSSION THREAD

**Discussion by danielsavinoff** [radix-ui/primitives#3026] [Help]
I already have used one popover in dropdown. There were a few issues where I had to use PopoverAnchor and close dropdown with the useState if popover was opened. Does renaming of one of two popovers would work? Because now neither of them opens up.

**Comment by b0yblake** [comment]
Normally, when the dropdown is closed by 'click-outside' or 'clicking an item', you can handle all these cases to disable the Popover (managed by the local state). However, the dropdown close action cannot manage the internal state of the Popover.

**Reply by b0yblake** [reply]
Popover maybe use the Portal or some kind of outside layout dropdown
